packages/cashier/src/stores/iframe-store.ts

```diff
@@ -1,8 +1,17 @@
 import { action, observable, makeObservable } from 'mobx';
 import Constants from 'Constants/constants';
+import { TRootStore } from 'Types';
+
+type TOnIframeLoadedCallback = (ev: MessageEvent) => void;
 
 export default class IframeStore {
-    constructor({ WS, root_store }) {
+    iframe_height = 0;
+    iframe_url = '';
+    is_session_timeout = true;
+    onIframeLoaded: TOnIframeLoadedCallback | null = null;
+    timeout_session: NodeJS.Timeout | null = null;
+
+    constructor(public root_store: TRootStore) {
```

Comment:
Doesn't need to define `WS` here?

Reply:
@hamid-deriv `WS` was unused in this class, I've just removed it.